### PR TITLE
Repo reorg fix maturin develop

### LINKED ISSUE #13: Repo reorg and fix maturin develop
Currently `maturin develop` fails to build desired wheel, need to reorganize the repo and fix `pyproject.toml`

### PR DESCRIPTION
**Pull request by newgnart**
To resolve #13

### CODE REVIEW

**Comment by leolara** [comment]
thanks @trangnv

**Comment by leolara** [comment]
@trangnv is it possible to move the tutorials to their own folder tutorial ?

**Comment by newgnart** [comment]
Of course they (.py and .ipynb files) should be movable wherever we want.

File: pyproject.toml [hunk 1]
```diff
@@ -13,5 +13,7 @@ classifiers = [
 
 
 [tool.maturin]
+bindings = 'pyo3'
 features = ["pyo3/extension-module"]
-python-source = "pychiquito"
+python-source = "python"
+module-name = "chiquito.rust_chiquito"
```
Comment (leolara) [inline pyproject.toml:19]: so this makes rust_chiquito a sub module of package chiquito?

**Review by leolara — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (newgnart) [reply pyproject.toml:19]: Right, an example using it [here](https://github.com/trangnv/chiquito/blob/65bb32dd651145ff294d03cc14b3548af9ff2ae9/python/chiquito/dsl.py#L4) 
In general, `chiquito.rust_chiquito` module has function listed [here](https://github.com/trangnv/chiquito/blob/65bb32dd651145ff294d03cc14b3548af9ff2ae9/src/lib.rs#L44)

Comment (leolara) [reply pyproject.toml:19]: Thanks, that makes much more sense structure wise

**Pull request merged**